shell/platform/android/io/flutter/embedding/FlutterView.java

```diff
@@ -38,1015 +33,781 @@
 import android.view.inputmethod.InputMethodManager;
 
 import org.json.JSONException;
-import org.json.JSONObject;
 
-import java.net.URI;
 import java.nio.ByteBuffer;
 import java.nio.ByteOrder;
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Locale;
 import java.util.Map;
-import java.util.concurrent.atomic.AtomicLong;
 
-import io.flutter.app.FlutterPluginRegistry;
-import io.flutter.plugin.common.ActivityLifecycleListener;
+import io.flutter.embedding.legacy.AccessibilityBridge;
+import io.flutter.embedding.legacy.TextInputPlugin;
 import io.flutter.plugin.common.BasicMessageChannel;
 import io.flutter.plugin.common.BinaryMessenger;
 import io.flutter.plugin.common.JSONMessageCodec;
 import io.flutter.plugin.common.JSONMethodCodec;
 import io.flutter.plugin.common.MethodChannel;
 import io.flutter.plugin.common.StandardMessageCodec;
-import io.flutter.plugin.common.StringCodec;
-import io.flutter.plugin.editing.TextInputPlugin;
-import io.flutter.plugin.platform.PlatformPlugin;
-import io.flutter.embedding.legacy.AccessibilityBridge;
-import io.flutter.view.FlutterMain;
-import io.flutter.embedding.legacy.FlutterNativeView;
-import io.flutter.view.FlutterRunArguments;
-import io.flutter.view.TextureRegistry;
 import io.flutter.view.VsyncWaiter;
 
 /**
- * An Android view containing a Flutter app.
+ * {@code View} which can render a Flutter UI by attaching itself to a {@link FlutterRenderer}.
```

Comment:
nit: Maybe worth mentioning that this is not only responsible for output (rendering) but also for input (touch events)